docs/geedocs/5.3.0/answer/7160006.html

```diff
@@ -78,6 +78,14 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
```

Comment:
would it be possible to push this change to github doc so the review/comparison with [5.2.5 release notes](https://www.opengee.org/geedocs/answer/7160006.html) be easier ?